alacritty/src/input/mod.rs

```diff
@@ -1045,6 +1070,7 @@ impl<T: EventListener, A: ActionContext<T>> Processor<T, A> {
         // If mouse mode is active, also look for bindings without shift.
         let fallback_allowed = mouse_mode && mods.contains(ModifiersState::SHIFT);
         let mut exact_match_found = false;
```

Comment:
```suggestion
        let mut match_found = false;
```
And drop the `fallback_match_found` not much value in having two of them.